src/components/templates/TemplateSpotDetail.vue

```diff
@@ -96,7 +96,11 @@
                             <tr v-if="spotDetails.UpdatedAt">
                                 <td>Updated At</td>
                                 <td>
-                                    {{ spotDetails.UpdatedAt.split('T')[0] }}
+                                    <AtomDatePicker
```

Comment:
It should show the updated at value by default
And should only show date picker when clicked on it.